brainz-mmd2-jaxb/src/main/java/org/musicbrainz/mmd2/Metadata.java

```diff
@@ -88,6 +89,7 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "", propOrder = {
+    "editNote",
```

Comment:
This probably doesn't matter in practice, but it might make more sense logically to have the edit-note element ordered after the list elements.